src/acts-api.js

```diff
@@ -2,40 +2,64 @@
 const SID_MAP = Object.fromEntries($store.pages.map((x, i) => [x.sID, i]));
 
 export function switchPage(sID) {
-  if (!sID in SID_MAP) {
-    console.error("Please input valid sID to switch to. Action aborted.");
-    return;
-  }
-  let currentPageNum = JSON.parse(JSON.stringify(SID_MAP[sID]));
+	if (!sID in SID_MAP) {
```

Comment:
Ok, what’s with the formatting? I thought we were just using config-less prettier?